lib/composables/useSorting/index.js

```diff
@@ -0,0 +1,47 @@
+import { ref, computed } from '@vue/composition-api';
+import _ from 'lodash';
+
+export const SORT_ORDER_ASC = 'asc';
+export const SORT_ORDER_DESC = 'desc';
+export const DATA_TYPE_STRING = 'string';
+export const DATA_TYPE_NUMERIC = 'numeric';
+export const DATA_TYPE_DATE = 'date';
+export const DATA_TYPE_OTHERS = 'others';
+
+export default function useSorting(headers, rows, useLocalSorting) {
+  const sortKey = ref(null);
+  const sortOrder = ref(null);
+
+  const sortedRows = computed(() => {
+    if (!useLocalSorting.value || sortKey.value === null) return rows.value;
```

Comment:
```suggestion
    if (!useLocalSorting.value || sortKey.value === null || sortOrder.value ===null) return rows.value;
```

**[required] I think we need yet another check here?** Seems that without it, `_.orderBy` below may potentially run with an invalid `null` order value. And it doesn't make sense to sort until we know in which order anyways.

I'm not entirely sure if we will actually run into any problems but it's quite common that `computed` runs before all values it depends on are ready and then it throws an unexpected error just to run correctly in another millisecond when the values are there, causing developers to investigate mysterious console errors that actually don't break anything for users :)

Reply:
It's always better to have double checks, than a potentially breaking code. Made the change.